src/pages/nft/claim/index.vue

```diff
@@ -709,6 +709,8 @@ export default {
     },
   },
   async mounted() {
+    this.claimingFreeEmail = this.walletEmail;
```

Comment:
set only if `free` is set? Not sure if we have such if case below for use

Reply:
Moved to `startFreePurchase()`

Reply:
Why did the watcher on `walletEmail` didn't work though

Reply:
my case is already logged in class page